unity-client/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AssetBundles/AB/AssetBundlesLoader.cs

```diff
@@ -8,9 +8,10 @@ namespace DCL
 {
     public class AssetBundlesLoader
     {
-        private const int SKIPPED_FRAMES_AFTER_BUDGET_TIME_FOR_NEARBY_ASSETS = 1;
-        private const int SKIPPED_FRAMES_AFTER_BUDGET_TIME_FOR_DISTANT_ASSETS = 10;
-        private const float MAX_SQR_DISTANCE_FOR_QUICK_LOADING = 5000f;
+        private const float MAX_LOAD_BUDGET_TIME = 0.004f;
+        private const int SKIPPED_FRAMES_AFTER_BUDGET_TIME_IS_REACHED_FOR_NEARBY_ASSETS = 1;
+        private const int SKIPPED_FRAMES_AFTER_BUDGET_TIME_IS_REACHED_FOR_DISTANT_ASSETS = 5;
+        private const float MAX_SQR_DISTANCE_FOR_QUICK_LOADING = 8000f;
```

Comment:
This distance is inside our loading radius, right? How are we relating it to that radius?

Reply:
For now `MAX_SQR_DISTANCE_FOR_QUICK_LOADING` is only an approximate value based on a distance measured by me on the world. If we have the loading radius accessible, I think it would be a good idea to base the `MAX_SQR_DISTANCE_FOR_QUICK_LOADING` value on it (for example: using a percentage of the radius value), what do you think?

Reply:
Yeah, the ideal is to adapt to the loading radius, but for now we can simply tune this value knowing on advance the radius in meters.

I think right now the loading radius is of 4 parcels (the unloading radius being 8, to prevent having loading/unloading overheads overlap).

So, each parcel is 16 meters. 16 * 4 =  64 meters. 64 squared = 4096. So the slow loading queue shouldn't never trigger with 8000?

Reply:
I have set the `MAX_SQR_DISTANCE_FOR_QUICK_LOADING` as `6000f` and I have added a temporal UI tool (it will be deleted after) in order to view how both queues are being filling in the build runtime.